src/components/language/__tests__/language.test.js

```diff
@@ -4,25 +4,17 @@ import Language from "../language";
 import { Game } from "../../../game/game";
 
  describe("Language",()=>{
+    var language;
+    var new_game;
+    beforeEach(() => {
+         new_game = new Game(null);
+        language = shallow(<Language game={new_game}  />);
+    });
      it("should render language component without fail",()=>{
          shallow(<Language/>)
      })
-     it("should change the language when a radio button is checked",()=>{
-        const new_game = new Game(null);
-        const component = shallow(<Language game={new_game} />);
-        component.find(".radio1").simulate("click",{target:{value:"english"}});
-        expect(new_game.language).toBe("english")
-    })
-    it("should change the language when a radio button is checked",()=>{
-        const new_game = new Game(null);
-        const component = shallow(<Language game={new_game} />);
-        component.find(".radio2").simulate("click",{target:{value:"english"}});
-        expect(new_game.language).toBe("english")
-    })
-    it("should change the language when a radio button is checked",()=>{
-        const new_game = new Game(null);
-        const component = shallow(<Language game={new_game} />);
-        component.find(".radio3").simulate("click",{target:{value:"english"}});
-        expect(new_game.language).toBe("english")
-    })
+     it("should render radio buttton componnt without fail", () => {
```

Comment:
Small typo `componnt ` should be `component`.

Reply:
done